src/main/kotlin/uk/gov/dluhc/printapi/service/PrintResponseFileService.kt

```diff
@@ -1,22 +1,27 @@
 package uk.gov.dluhc.printapi.service
 
+import com.fasterxml.jackson.databind.ObjectMapper
 import mu.KotlinLogging
 import org.springframework.stereotype.Service
+import uk.gov.dluhc.printapi.printprovider.models.PrintResponses
 
 private val logger = KotlinLogging.logger {}
 
 @Service
 class PrintResponseFileService(
     val sftpService: SftpService,
+    val objectMapper: ObjectMapper,
+    val printResponseProcessingService: PrintResponseProcessingService
 ) {
     fun processPrintResponseFile(directory: String, fileName: String) {
         val printResponsesString = sftpService.fetchFileFromOutBoundDirectory(directory, fileName)
-        processPrintResponseContent(printResponsesString)
+        val printResponses = parsePrintResponseContent(printResponsesString)
+        printResponseProcessingService.processBatchAndPrintResponses(printResponses)
         sftpService.removeFileFromOutBoundDirectory(directory, fileName)
     }
 
-    private fun processPrintResponseContent(printResponsesString: String) {
+    private fun parsePrintResponseContent(printResponsesString: String): PrintResponses {
         logger.debug { "processing $printResponsesString" }
```

Comment:
Minor suggestion, @fayimora . 
It might be better to change `processing` to `parsing` in the logger for consistency with the method name.